modules/mobile/app/controllers/mobile/v0/immunizations_controller.rb

```diff
@@ -9,6 +9,17 @@ def index
         render json: Mobile::V0::ImmunizationSerializer.new(immunizations_adapter.parse(service.get_immunizations))
       end
 
+      def location
+        lh_location = service.get_location(params[:id])
```

Comment:
I think we'll never get `nil` for a location as not finding one should return a 404. Any chance walking the data (`[:identifier][0][:value]`) would end up with a nil value?

Reply:
My guess would be no since not found is a 404